src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -47,7 +47,15 @@ public class SortCommandParser implements Parser<SortCommand> {
     @Override
     public SortCommand parse(String args) throws ParseException {
         requireNonNull(args);
-
+        if (args.trim().equals("index")) {
```

Comment:
I’m a bit unsure on having users type “index” cos i feel that sort on its own should just sort by index.

Reply:
Because we want to make the index the ultimate tie breaker.